molecule/default/molecule.yml

```diff
@@ -4,36 +4,30 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: .yamllint.yml
+lint: |
+  set -eux
+  yamllint -c .yamllint.yml .
+  ansible-lint -v
+  flake8
 platforms:
   - name: centos-6
     image: docker.io/linuxsystemroles/centos-6
     privileged: true
   - name: centos-7
-    image: docker.io/linuxsystemroles/centos-7
+    image: centos/systemd
```

Comment:
Is it correct not to specify a tag here?

Reply:
Apparently the only tag available is `latest` (which is the default if no tag is specified) - https://registry.centos.org/centos/systemd